protostar/composition_root.py

```diff
@@ -152,6 +151,7 @@ def build_di_container(script_root: Path):
             migrator_builder=Migrator.Builder(
                 migrator_execution_environment_builder=MigratorExecutionEnvironment.Builder(),
                 gateway_facade_builder=GatewayFacade.Builder(project_root_path),
+                cwd=project_root_path,
```

Comment:
looks like a dead argument?

Reply:
It's not dead. `cwd` was maybe a bad name for it.
Resolving: renamed to `project_root_path`.